<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connections</class>
 <widget class="qMRMLWidget" name="Connections">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>587</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="receive_port_widget"/>
   </item>
   <item>
    <widget class="QLineEdit" name="send_port_widget"/>
   </item>
   <item>
    <widget class="QPushButton" name="connect_button">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="disconnect_button">
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
